test/test.dart

```diff
@@ -340,7 +340,7 @@ const int jsMaxInt = 9007199254740991;
 const int jsMinInt = -9007199254740991;
 
 //Overrides test method so we can filter tests
-void test(String name, dynamic Function() testFunction, {dynamic skip}) {
+void test(String name, dynamic Function() testFunction, {dynamic skip, Map<String, dynamic>? onPlatform}) {
```

Comment:
We don't use this parameter anywhere `onPlatform`. Why do we need it?

Reply:
that is just making the interface compatible with the testing framework API. In-case we need it.